src/main/sensors/gyro.c

```diff
@@ -798,7 +799,7 @@ static FAST_CODE void gyroUpdateSensor(gyroSensor_t *gyroSensor, timeUs_t curren
     if (gyroDebugMode == DEBUG_NONE) {
         for (int axis = 0; axis < XYZ_AXIS_COUNT; axis++) {
             // NOTE: this branch optimized for when there is no gyro debugging, ensure it is kept in step with non-optimized branch
-            float gyroADCf = (float)gyroSensor->gyroDev.gyroADC[axis] * gyroSensor->gyroDev.scale;
+            float gyroADCf = ((float)gyroSensor->gyroDev.gyroADC[axis] * gyroSensor->gyroDev.scale) + ((float)gyroConfig()->gyro_offset)/1000;
```

Comment:
gyroConfig()->gyro_offset / 1000 should be precomputed in the init function

Reply:
i dont understand what gyroZero and a data type has to do with a precompution for gyroConfig()->gyro_offset / 1000

just add in line 90: static float gyro_offset;
and init function gets a gyro_offset = gyroConfig()->gyro_offset / 1000

float division should not happen in the crictial path, even more if they can be reduced that easy.